### PR TITLE
Pic 410 add r53

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**
Added route 53 as per user guide. Please check carefully :-)

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+}
+
+resource "aws_route53_record" "add_cname_email" {
+  name    = var.zone-name
+  zone_id = aws_route53_zone.example_team_route53_zone.zone_id
+  type    = "CNAME"
+  records = ["test.org"]
+  ttl     = "300"
+}
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:30]: i believe you don't need this- not a blocker though

Comment (chris-faulkner-mojdt) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:30]: And it won't hurt ?

Comment (mtrbls) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:30]: It'd be better to delete it

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
+resource "aws_route53_zone" "example_team_route53_zone" {
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:1]: Could you rename your resource please. `example_team` is not a suitable name

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf]: Done

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf (hunk 2 above)
Comment (mtrbls) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:1]: and all references below

Comment (chris-faulkner-mojdt) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:1]: As soon as I change it it goes red in my IDE...

Comment (chris-faulkner-mojdt) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf:1]: Suggest the web page puts that in square brackets to denote a required change

**Pull request merged**